domapping/mapping.py

```diff
@@ -312,6 +316,24 @@ def __gen_type_properties(json_schema, path, resolver, config, es_mapping):
     return es_mapping
 
 
+def __guess_enum_type(enum_array, path):
```

Comment:
Thanks a lot @nharraud. I was testing the tool right now and the result for this file [CMS Questionnaire](https://github.com/cernanalysispreservation/analysis-preservation.cern.ch/blob/master/cap/modules/experiments/static/jsonschemas/cms/cms_questionnaire_schema.json). I needed to remove ``aditionalProperties`` from the schema (Only two lines out of 1200). The result looks like:
```json
{
    "date_detection": true,
    "_all": {
        "enable": true
    },
    "properties": {
        "general": {
            "type": "object",
            "properties": {
                "faq_awareness": {
                    "type": "string"
                },
                "meetings_awareness": {
                    "type": "string"
                },
                "forum_awareness": {
                    "type": "string"
                },
                "forum_subscription": {
                    "type": "string"
                },
                "past_interaction": {
                    "type": "string"
                }
            }
        },
        "multivariate": {
            "type": "object",
            "properties": {
                "correlation_studies": {
                    "type": "object",
                    "properties": {
                        "name": {
                            "type": "string"
                        },
                        "comments": {
                            "type": "string"
                        }
                    }
                },
(...)
                "usage": {
                    "type": "string"
                },
                "bcgrnd_shape": {
                    "type": "object",
                    "properties": {
                        "name": {
                            "type": "string"
                        },
                        "comments": {
                            "type": "string"
                        }
                    }
                }
            }
        },
        "feedback": {
            "type": "object",
            "properties": {
                "subject_to_change": {
                    "type": "object",
                    "properties": {
                        "name": {
                            "type": "string"
                        },
                        "comments": {
                            "type": "string"
                        }
                    }
                },
                "feedback": {
                    "type": "string"
                },
                "additional_comments": {
                    "type": "string"
                }
            }
        }
    },
    "numeric_detection": true
}
```